README.md

```diff
@@ -1,424 +1,204 @@
-my_axon.attach(
-    forward_fn = forward_my_synapse, 
-    verify_fn=verify_my_synapse,
-    blacklist_fn = blacklist_my_synapse,
-    priority_fn = prioritize_my_synape
-).start()
-```     
-
-Dendrite: Represents the abstracted implementation of a network client module
-designed to send requests to those endpoints to receive inputs.
-
-Example:
-```python
-dendrite_obj = dendrite( wallet = bittensor.wallet() )
-# pings the axon endpoint
-await d( <axon> )
-# ping multiple axon endpoints
-await d( [<axons>] ) 
-# Send custom synapse request to axon.
-await d( bittensor.axon(), bittensor.Synapse() ) 
-# Query all metagraph objects.
-await d( meta.axons, bittensor.Synapse() ) 
+pip install .
 ```
 
-## Setting weights on root network
-Use the `root` subcommand to access setting weights on the network across subnets.
+---
 
-```bash
-btcli root weights --wallet.name <coldname> --wallet.hotkey <hotname>
-Enter netuids (e.g. 0, 1, 2 ...):
-# Here enter your selected netuids to set weights on
-1, 2
+## Install on Windows
 
->Enter weights (e.g. 0.09, 0.09, 0.09 ...): 
-# These do not need to sum to 1, we do normalization on the backend.
-# Values must be > 0
-0.5, 10
+To install and run Bittensor SDK on Windows you must install [**WSL 2** (Windows Subsystem for Linux)](https://learn.microsoft.com/en-us/windows/wsl/about) on Windows and select [Ubuntu Linux distribution](https://github.com/ubuntu/WSL/blob/main/docs/guides/install-ubuntu-wsl2.md). 
 
-Normalized weights: 
-        tensor([ 0.5000, 10.0000]) -> tensor([0.0476, 0.9524])
+After you installed the above, follow the same installation steps described above in [Install on macOS and Linux](#install-on-macos-and-linux).
 
-Do you want to set the following root weights?:
-  weights: tensor([0.0476, 0.9524])
-  uids: tensor([1, 2])? [y/n]: 
-y
+**ALERT**: **Limited support on Windows**
+While wallet transactions like delegating, transfer, registering, staking can be performed on a Windows machine using WSL 2, the mining and validating operations are not recommended and are not supported on Windows machines.
 
-⠏ 📡 Setting root weights on test ...
-```
+---
 
-## Bittensor Subnets API
+## Verify the installation
 
-This guide provides instructions on how to extend the Bittensor Subnets API, a powerful interface for interacting with the Bittensor network across subnets. The Bittensor Subnets API facilitates querying across any subnet that has exposed API endpoints to unlock utility of the Bittensor decentralized network.
+You can verify your installation in either of the two ways as shown below:
 
-The Bittensor Subnets API consists of abstract classes and a registry system to dynamically handle API interactions. It allows developers to implement custom logic for storing and retrieving data, while also providing a straightforward way for end users to interact with these functionalities.
+### Verify using the `btcli` command
```

Comment:
The default SDK installation does not include `btcli`. To install `btcli` user need to run the command:
```bash
pip install bittensor[btcli]
```
Even in this case, the command `btcli --help` will show the `btcli` version, but not `bittensor sdk`.
To check the `btsdk` version, user need to run the command:
```bash
python -m bittensor
```
This can be mentioned below in the step `Verify using Python interpreter` or/and `Verify the installation`.

Reply:
The community has mentioned many times that they don't want to install a lot of unnecessary dependencies for certain tasks.
I would like to see a section of the documentation where we explicitly highlight this change.

Currently, the user can install the bittensor with several options:
- `pip install bittensor` - only SDK bittensor;
- `pip install bittensor[btcli]` - SDK bittensor with `btcli` (btcli will be installed as an independent tool. Python package name is `bittensor-cli`);
- `pip install bittensor[torch]` - Torch package will be installed too;
- `pip install bittensor[cubit]` - Cubit package will be installed too.